src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Square extends Figure {
```

Comment:
The name 'Square' is appropriate as it is a noun, not a verb, which follows the guideline for class naming.